Automattic/jetpack
# Search Dashboard: Updating PlanUsageSection to more closely match design

Fixes #26781 

#### Changes proposed in this Pull Request:


- Adds callbacks to the UsageMeters component.
- Adds props to DonutMeterContainer so we can pass in the callbacks.
- Fixes a wrapping issue in the DonutMeterContainer.
- Removes the "View details" link that was initially meant to show/hide the Record Meter bar.
- Adds icon indicating external support link.
- Adds special case for handling "unlimited" requests.

#### Other information:


#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



No

#### Testing instructions:






1. Visit any site with a Jetpack Search plan.
2. Open the Search dashboard and append `&new_pricing_202208=1` to the URL.
3. Confirm the meters show correctly with the "i" icons.
4. Confirm the "View details" link is no longer present.

Before:
![WrapBefore](https://user-images.githubusercontent.com/40267301/195273999-5b2f7082-d2e8-4b36-a81d-80f0d7ba71e9.png)

After:
![After2](https://user-images.githubusercontent.com/40267301/195283615-47f2eb6d-cbed-4d46-89c0-97ce87532090.png)
